config/dev.exs

```diff
@@ -1,7 +1,7 @@
 import Config
 
 config :skate,
-  tileset_url: "https://mbta-map-tiles-dev.s3.amazonaws.com/osm_tiles",
+  tileset_url: "https://mbta-map-tiles-dev.s3.amazonaws.com/skate_osm_tiles",
```

Comment:
praise: Thank you for changing this -- much nicer to see the tiles locally match what is in prod!